README.md

```diff
@@ -1,3 +1,65 @@
-# file-viewer
+## file-viewer
 
-A rewrite of react-file-viewer
+Supported file formats:
+- bmp
+- csv
+- docx
+- gif
+- jpeg
+- jpg
+- pdf
+- png
+- xlsx
+
+### Install
+```
+# via npm
+npm install @resolve_ch/file-viewer
+```
+### FileViewer example
+```
+import React from 'react';
+import ReactDOM from 'react-dom';
+import FileViewer from '@resolve_ch/file-viewer';
+
+class App extends React.Component {
+  render() {
+    return (
+      <FileViewer
+        filePath="https://upload.wikimedia.org/wikipedia/en/a/a9/Example.jpg"
+      </FileViewer>
+    );
+  }
+}
+
+ReactDOM.render(<App/>, document.body);
+```
+### FileViewer props
+```
+{
+  // The url of the resource to be shown by the FileViewer.
+  filePath: string,
+  
+  // Type of resource to be shown (one of the supported file formats, eg 'png').
+  // Passing in an unsupported file type will result in displaying an unsupported file type message (or a custom component).
+  // By default, the extension of the filePath is used to determine the fileType, but it can be overwritten using this prop.
+  fileType: string [optional]
+  
+  // Can be used to extend the file-viwer or overwrite the existing drivers, eg: '{ mp4: VideoPlayerViewer }'
+  drivers: object [optional]
+
+  // Called when an error is thrown due to fetching or rendering problems
+  onError: function [optional]
```

Comment:
Can you maybe add the list of arguments passed to this function, as well as the props passed to the 2 components below?